src/main/java/org/opensearch/timeseries/util/ParseUtils.java

```diff
@@ -715,44 +712,22 @@ public static boolean shouldUseResourceAuthz(Settings settings) {
 
     /**
      * Verifies whether the user has permission to access the resource.
-     * @param requestedUser user from request
-     * @param configIndex index where config is stored
-     * @param configId config id
      * @param shouldEvaluateWithNewAuthz true only if resource-sharing feature and filter_by_backend role, both are enabled.
-     * @param listener action listener
      * @param onSuccess consumer function to execute if user has permission
      * @param successArgs arguments to pass to the consumer function
      * @param fallbackOn501 consumer function to execute if user does not have permission
      * @param fallbackArgs arguments to pass to the consumer function
      */
     public static void verifyResourceAccessAndProcessRequest(
-        User requestedUser,
-        String configIndex,
-        String configId,
         boolean shouldEvaluateWithNewAuthz,
-        ActionListener<? extends ActionResponse> listener,
         Consumer<Object[]> onSuccess,
         Object[] successArgs,
         Consumer<Object[]> fallbackOn501,
         Object[] fallbackArgs
     ) {
-        // TODO: Remove this feature flag check once feature is GA, as it will be enabled by default
-        // detectorId will be null when this is a create request and so we don't need resource authz check
-        if (shouldEvaluateWithNewAuthz && !Strings.isNullOrEmpty(configId)) {
-            ResourceSharingClient resourceSharingClient = ResourceSharingClientAccessor.getInstance().getResourceSharingClient();
-            resourceSharingClient.verifyAccess(configId, configIndex, ActionListener.wrap(isAuthorized -> {
```

Comment:
with security plugin's lates change, access will be auto evaluated. Thus, we no longer need to explicitly call verifyAccess.

Reply:
Okay so from my understanding from offline conversation:
verifyAccess will look for a few things currently: 

- global flag must be turned on along with ad flag
- request must be of DocRequest type
- any existing sharing info must have been migrated to the new model using /migrate  API offered by security plugin